akka-bench-jmh/src/main/scala/akka/stream/GraphBuilderBenchmark.scala

```diff
@@ -16,22 +17,22 @@ class GraphBuilderBenchmark {
   var complexity = 0
 
   @Benchmark
-  def flow_with_map(): Unit = {
-    MaterializationBenchmark.flowWithMapBuilder(complexity)
+  def flow_with_map(blackhole: Blackhole): Unit = {
+    blackhole.consume(MaterializationBenchmark.flowWithMapBuilder(complexity))
```

Comment:
isn't it enough to expose the return type in the method? AFAIK that is equivalent to this (JMH should blackhole it automagically).